application/src/main/java/org/opentripplanner/routing/api/request/preference/WalkPreferences.java

```diff
@@ -92,7 +92,10 @@ public int boardCost() {
     return boardCost.toSeconds();
   }
 
-  /** Used instead of walk reluctance for stairs */
+  /**
+   * Used on top of reluctance for stairs. If the value is set to 1, walking on stairs is treated
+   * the same as walking on flat ground.
+   */
```

Comment:
```suggestion
  /**
   * Used on top of walk reluctance for stairs. If the value is set to 1, walking on stairs is
   * treated the same as walking on flat ground.
   */
```

Reply:
It's a bit weird that the default walk and stairs reluctance have the same value. Therefore, I'm ok with not adjusting the default stairs reluctance down in this class.